src/components/Attachment/AttachmentActions.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Text, TouchableOpacity } from 'react-native';
-import styled from '@stream-io/styled-components';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
```

Comment:
shouldn't all of these be like this
```suggestion
import styled from 'styled-components/native';
```